pages/searchResults.page.js

```diff
@@ -0,0 +1,41 @@
+let BasePage = require("../base/base.page");
+
+let { WebButton } = require("../elements/button.element");
+let { WebView } = require("../elements/view.element");
+
+let searchResultsTextBoxLocator = by.xpath('//*[@class="heading-counter"]');
+let firstSearchResultBtn = by.css('.row > li:first-child h5 > a');
+let baseElementLocator = by.xpath('//*[@class="navigation_page" and text()="Search"]');
+
+class SearchResultsPage extends BasePage {
+
+    async searchResults() {
+        return await this.getsSearchResultsTextBox().getText();
+    }
+
+    async searchResultsCount() {
+        let stringvalue = await this.searchResults();
+        return parseInt(stringvalue, 10);
+    }
+
+
+    async selectFirstResult() {
```

Comment:
try to use locator like `(//div[@class = "product-container"]//a[@class = "product-name"])[INDEX]` and based on it, create method which can find and click any product in list using index